pkg/kubelet/client/kubelet_client_test.go

```diff
@@ -19,14 +19,9 @@ package client
 import (
 	"testing"
 
-	v1core "k8s.io/client-go/kubernetes/typed/core/v1"
 	restclient "k8s.io/client-go/rest"
 )
 
-// Ensure a node client can be used as a NodeGetter.
-// This allows anyone with a node client to easily construct a NewNodeConnectionInfoGetter.
-var _ = NodeGetter(v1core.NodeInterface(nil))
```

Comment:
leave the assertion somewhere

Reply:
this assertion is not valid nomore b/c the client doesn't receive ctx as parameter.